package.json

```diff
@@ -17,6 +17,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^1.1.1",
```

Comment:
We don't need any extra library for the icons as https://react-icons.github.io/react-icons has already bunch of icons.